cmd/cert.go

```diff
@@ -121,15 +121,13 @@ func runCert(_ context.Context, c *cli.Command) error {
 		err = fmt.Errorf("Unrecognized elliptic curve: %q", c.String("ecdsa-curve"))
 	}
 	if err != nil {
-		// log.Fatalf("Failed to generate private key: %v", err)
-		return err
+		return fmt.Errorf("Failed to generate private key: %w", err)
```

Comment:
Actually here and other places: Error strings should not be capitalized https://go.dev/wiki/CodeReviewComments#error-strings

It's not a blocker but indeed what Golang standard recommends.

Reply:
Introducing lint changes is not in this PR's scope IMO